Sources/SwiftGD/Format.swift

```diff
@@ -116,7 +116,7 @@ extension LibGdParametrizableExportFormatter {
 private struct BMPFormatter: LibGdParametrizableFormatter {
     /// The parameters to apply on exports
     fileprivate var exportParameters: Int32
-
+    
```

Comment:
Nit: don’t change line formatting

Reply:
Yes, I apologize; I'm used to using Xcode's `Ctrl+I` key. I've corrected all code format please check.